### PR TITLE
feat(AAiT-web-2): Add success-stories

### PR DESCRIPTION
**Pull request by kaleabe-n**
Screenshot 2023-08-24 102821
Screenshot 2023-08-24 102813
Screenshot 2023-08-24 102805
Screenshot 2023-08-24 102755
6e2bf)

### CODE REVIEW

File: aait/web/group-2/components/footer/index.tsx
Comment (FevenBelay23) [inline aait/web/group-2/components/footer/index.tsx:1]: use PascalCase naming for components

File: aait/web/group-2/public/images/Google.png
Comment (FevenBelay23) [inline aait/web/group-2/public/images/Google.png:1]: use kebab-case  naming style for all files and folders other than components.

File: aait/web/group-2/app/success-stories/page.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+"use client"
+
+import SingleSuccessStory from "@/components/SingleSuccessStory";
+import { useGetSuccessStoriesQuery } from "@/store/features/successStories";
+import { SuccessStory } from "@/types/successStories";
+
+const SuccessStories = () => {
+    const {isLoading,isSuccess,data:successStories,error,isError} = useGetSuccessStoriesQuery()
+    return ( <main className="">
```
Comment (FevenBelay23) [inline aait/web/group-2/app/success-stories/page.tsx:9]: use the extension prettier to make your code properly structured.

**Review by FevenBelay23 — CHANGES_REQUESTED** [review]

File: aait/web/group-2/app/success-stories/page.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+          Behind every success is a story. Learn about the stories of A2SVians
+        </p>
+        <div className="w-20 h-2 bg-primary mt-4"></div>
+      </section>
+      {isLoading && (
+        <div className="text-center text-6xl my-48 font-poppins">
+          loading...
+        </div>
+      )}
+      {isSuccess && !isError && (
```
Comment (Aklile-Yilma) [inline aait/web/group-2/app/success-stories/page.tsx:31]: IsSuccess tells you if its successful why use isError to?

File: aait/web/group-2/components/footer/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            <Link href={"#"}>Terms of service</Link> |{" "}
+            <Link href={"#"}>Privacy Policy</Link>
+          </p>
+        </div>
+        <div>
+          <h3 className="text-gray-800 font-semibold mt-6 text-xl md:mt-3 mb-3">
+            Our teams
+          </h3>
+          <ul>
+            <li className="mb-2">Advisors</li>
```
Comment (Aklile-Yilma) [inline aait/web/group-2/components/footer/Footer.tsx:24]: You can take this out to a json object and map through it to get this ui

File: aait/web/group-2/components/single-success-story/SingleSuccessStory.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+          </p>
+          <p className="text-white font-sans text-sm sm:text-normal md:text-2xl font-semibold m-3">
+            {successStory.location}
+          </p>
+        </div>
+      </div>
+      <div className="w-4/5 md:w-1/2 mt-10">
+        <div className="flex flex-col pt-8 ">
+          {successStory.story.map((story) => {
+            return (
```
Comment (Aklile-Yilma) [inline aait/web/group-2/components/single-success-story/SingleSuccessStory.tsx:28]: You can remove the return statement by replacing the curly brace with brackets

**Review by Aklile-Yilma — APPROVED** [review]
LGTM

**Review by FevenBelay23 — APPROVED** [review]

**Review by FevenBelay23 — APPROVED** [review]

**Review by zelalemyetsedaw — APPROVED** [review]

**Review by FevenBelay23 — APPROVED** [review]

**Pull request merged**